Detections/MultipleDataSources/UserImpersonateByRiskyUser.yaml

```diff
@@ -0,0 +1,69 @@
+id: f4a28082-2808-4783-9736-33c1ae117475

```

Comment:
Could we do another query with the reverse here as well, risky AWS signin seen in Azure?

Reply:
AWS doesn't have any Risky Signin signals